lib/cocoapods/target/build_settings.rb

```diff
@@ -424,6 +429,22 @@ def merged_xcconfigs(xcconfig_values_by_consumer_by_key, attribute)
         end
       end
 
+      # Merges the spec-defined xcconfig into the derived xcconfig,
+      # overriding any singular settings and merging plural settings.
+      #
+      # @param  [Hash<String,String>] spec_xcconfig_hash the merged xcconfig defined in the spec.
+      #
```

Comment:
also nit remove

Reply:
remove what?

Reply:
I meant the extra line between `@param`

Reply:
In this repo, there's normally a blank line in between yard directives